### DISCUSSION TITLE
Request to Add SCSS Support in LazyVim Extras

### DISCUSSION THREAD

**Discussion by Muzammil-hasan** [LazyVim/LazyVim#3572] [Extra Requests]
Hi Folke,

Could we add SCSS support to LazyVim extras? It would greatly help those of us working with SCSS.

Thanks for considering this!

**Comment by folke** [comment]
Just install `cssls` through mason and install the scss treesitter parser with `:TSInstall scss`. 
No configuration needed.

You can also enable the prettier extra to format your scss files

**Reply by snatvb** [reply]
maybe it's better? https://github.com/wkillerud/some-sass
I'd used it before I migrated to lazyvim and it works pretty cool
